File: BusReservationApi/Models/Passenger.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace BusReservationApi.Models
{
    public partial class Passenger
    {
        public int Pid { get; set; }
        public int Tid { get; set; }
        public string Pname { get; set; }
        public int Age { get; set; }
        public string Adhaar { get; set; }
        public string Gender { get; set; }

        public virtual TransactionDetail TidNavigation { get; set; }
    }
}

File: frontend/src/app/passenger.ts
export class Passenger {
    pId: number 
    tId: number
    pName: string
    age: number
    adhaar: string
    gender: string

    constructor(pid: number, tid:number, pname:string, age:number, adhar:string, gender:string) {
        this.pId = pid
        this.tId = tid
        this.age = age
        this.adhaar = adhar
        this.gender = gender
        this.pName = pname
    }
}
